docs/UserGuide.md

```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding an employee: `addE`
 
-Adds a person to the address book.
+Adds an employee to the employees list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `addE n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+An employee can have any number of tags (including 0)
 </div>
 
+Acceptable values for each parameter:
+* Name: John Doe / Alex / A. P. Mary
+* Phone number: +65 9123 4567 / 62353535 
+* Email: john.doe@gmail.com 
+* Address: Any string resembling a location
+* Tag (optional): Any string
```

Comment:
We do not have the phrase `Acceptable values for each parameter` in findE and addP. Maybe we could refactor this part to be expressed in sentences for consistency.

Reply:
I'll remove it then, since other commands do not have this.